packages/modules/web_themes/colors/source/src/assets/js/processMessages.ts

```diff
@@ -35,6 +37,7 @@ const topicsToSubscribe = [
 	'openWB/optional/et/#',
 	'openWB/system/#',
 	'openWB/LegacySmartHome/#',
+	'openWB/command/#',
```

Comment:
Bitte nicht den kompletten Baum abonnieren. Relevant sind nur:
1. globale Systemmeldungen unter `openWB/system/messages/+`
2. spezifische Nachrichten für den Client unter `openWB/command/<mqttClientId>/messages/+`
3. und ein veraltetes Topic unter `openWB/command/<mqttClientId>/error`
Ich wüsste aktuell jedoch keine Benutzeraktion, die eine Nachricht vom Backend an den Client im Colors Theme auslöst. Dort werden ja keine grundlegenden Konfigurationen gemacht, die Feedback erfordern. Demnach würde eigentlich das 1. Topic ausreichen.

Reply:
Ich hatte kürzlich den Fall, dass das Auswählen des Tages-Graphen per mqtt fehlschlug. Im Standard-UI erschien dann ein Popup mit der Fehlermeldung von der openwb. Diese kam über /^openWB\/command\/<mqttClientID>\/error$/i . Ich möchte solche Fehlermeldungen zumindest in der console anzeigen, damit die Fehlersuche erleichtert wird. Wenn dieser Topic veraltet ist, gibt es einen neuen für Fehlermeldungen? ist das system/messages?

Reply:
Ich habe die Subscription für die command-Topics rausgenommen

Reply:
Du kannst das command Topic gerne abonnieren, aber bitte nur für Deine client-id. Ansonsten landen auch alle Nachrichten von anderen Nutzern im Theme, was nur für Verwirrung sorgt.

Reply:
Ok, danke. Ich habe die Subscription so geändert, dass nur für den aktuellen Client abonniert wird.